nav2_map_server/test/component/CMakeLists.txt

```diff
@@ -1,54 +1,113 @@
 include_directories(${PROJECT_SOURCE_DIR}/test)
 
 # map_server component test
-ament_add_gtest_executable(test_map_server_node
-  test_map_server_node.cpp
+ament_add_gtest_executable(test_map_server_2d_node
+    test_map_server_2d_node.cpp
   ${PROJECT_SOURCE_DIR}/test/test_constants.cpp
 )
-ament_target_dependencies(test_map_server_node rclcpp nav_msgs)
-target_link_libraries(test_map_server_node
+ament_target_dependencies(test_map_server_2d_node rclcpp nav_msgs nav2_msgs nav2_util)
+target_link_libraries(test_map_server_2d_node
+    ${map_io_library_name}_2d
```

Comment:
Why do we need to link over MapIO library?

Reply:
No need with `server test`! removed them.